patches/net/minecraft/data/models/BlockModelGenerators.java.patch

```diff
@@ -64,23 +64,25 @@
 +    }
 +
 +    @org.jetbrains.annotations.Nullable public final net.neoforged.neoforge.common.data.ExistingFileHelper fileHelper;
++    public final String modId;
 +
      public BlockModelGenerators(
 -        Consumer<BlockStateGenerator> p_124481_, BiConsumer<ResourceLocation, Supplier<JsonElement>> p_124482_, Consumer<Item> p_124483_
```

Comment:
Switch around the order of the constructors to make the patch a bit more readable